### PR TITLE
NearMe Header

### PR DESCRIPTION
**Pull request by sarahabraham014**
# Closes/Fixes/Resolves

###Fixes 40

# Explain the feature

Added heading with search and bell button
and wrappers in NearMe

## Does this PR introduce a breaking change

No

## Screenshots

React App - Google Chrome 30-01-2021 22_01_22 (2)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -14,22 +14,22 @@ appearance, race, religion, or sexual identity and orientation.
 Examples of behavior that contributes to creating a positive environment
```
Comment (rejahrehim) [inline CODE_OF_CONDUCT.md:14]: Don't include unrelated changes in the PR. Remove this changes in this PR. Better send another

File: package.json [hunk 2]
```diff
@@ -1,4 +1,56 @@
 {
+<<<<<<< HEAD
```
Comment (rejahrehim) [inline package.json:2]: Conflict not resolved properly

File: package.json [hunk 3]
```diff
@@ -1,4 +1,56 @@
...
+        "development": [
+            "last 1 chrome version",
+            "last 1 firefox version",
+            "last 1 safari version"
+        ]
+    },
+    "devDependencies": {
+        "prettier": "2.2.1"
+    }
+=======
```
Comment (rejahrehim) [inline package.json:53]: Conflict not resolved properly

File: package.json [hunk 4]
```diff
@@ -56,4 +108,5 @@
       "pre-push": "npm run build"
     }
   }
+>>>>>>> 57e6c5193b1f22b5a6497fcf2c81af6e970085f9
```
Comment (rejahrehim) [inline package.json:111]: Conflict not resolved properly

**Review by rejahrehim — CHANGES_REQUESTED** [review]

File: src/components/NearMe/index.js [hunk 5]
```diff
@@ -1,5 +1,17 @@
...
 export default function NearMe() {
-    return <div>Near me</div>;
+    return (
+        <div>
+            <Topbar/>
+        </div>
+     
+        
+
+
```
Comment (riginoommen) [inline src/components/NearMe/index.js:12]:
```suggestion
```

File: src/components/NearMe/top.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+            Wastebins
+             </a>
+             <a href="" class="border-2 border-green-500 rounded-full font-bold text-green-500 px-4 py-3 transition duration-300 ease-in-out hover:bg-green-500 hover:text-white mr-6 text-base">
+            Restrooms
+             </a>
+             
+        </div>
+
+        
+
```
Comment (riginoommen) [inline src/components/NearMe/top.js:65]:
```suggestion
        </div>
```

File: src/router/PublicRouter.js [hunk 7]
```diff
@@ -5,6 +5,7 @@ import Destination from "../components/Destination/index";
 import NavBar from "../components/Common/NavBar";
 import Experiences from "../components/Experiences/Experiences";
 
+
```
Comment (riginoommen) [inline src/router/PublicRouter.js:8]:
```suggestion
```

File: src/components/NearMe/top.js (hunk 6 above)
Comment (riginoommen) [inline src/components/NearMe/top.js:65]:
```suggestion
```

File: src/components/NearMe/top.js [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+            Restrooms
+             </a>
+             
+        </div>
+        </div>
+    </div>   
+        
+            
+
+
```
Comment (riginoommen) [inline src/components/NearMe/top.js:68]:
```suggestion
```

File: public/index.html [hunk 9]
```diff
@@ -1,21 +1,21 @@
...
+        <meta charset="utf-8" />
+        <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="theme-color" content="#000000" />
+        <meta
+            name="description"
+            content="Web site created using create-react-app"
+        />
+        <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+        <!--
```
Comment (riginoommen) [inline public/index.html:13]:
```suggestion
<head>
    <meta charset="utf-8" />
    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="theme-color" content="#000000" />
    <meta
      name="description"
      content="Web site created using create-react-app"
    />
    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
    <!--
```

File: public/index.html [hunk 10]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
```
Comment (riginoommen) [inline public/index.html:3]:
```suggestion
 <head>
```

File: public/index.html [hunk 11]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+ <head>
```
Comment (riginoommen) [inline public/index.html:3]:
```suggestion
  <head```

File: public/index.html [hunk 12]
```diff
@@ -14,8 +14,8 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
+        <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+        <!--
```
Comment (riginoommen) [inline public/index.html:18]:
```suggestion
    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
    <!--
```

File: public/index.html [hunk 13]
```diff
@@ -24,12 +24,12 @@
...
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+        <title>React App</title>
+    </head>
+    <body>
+        <noscript>You need to enable JavaScript to run this app.</noscript>
+        <div id="root"></div>
+        <!--
```
Comment (riginoommen) [inline public/index.html:32]:
```suggestion
    <title>React App</title>
  </head>
  <body>
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="root"></div>
    <!--
```

File: public/index.html [hunk 14]
```diff
@@ -38,6 +38,5 @@
 
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
-    -->
-  </body>
+    --></body>
```
Comment (riginoommen) [inline public/index.html:41]:
```suggestion
    -->
  </body>
```

File: public/manifest.json [hunk 15]
```diff
@@ -1,25 +1,25 @@
...
+        {
+            "src": "logo512.png",
+            "type": "image/png",
+            "sizes": "512x512"
+        }
+    ],
+    "start_url": ".",
+    "display": "standalone",
+    "theme_color": "#000000",
+    "background_color": "#ffffff"
```
Comment (riginoommen) [inline public/manifest.json:24]:
```suggestion
    "name": "Create React App Sample",
    "icons": [
        {
            "src": "favicon.ico",
            "sizes": "64x64 32x32 24x24 16x16",
            "type": "image/x-icon"
        },
        {
            "src": "logo192.png",
            "type": "image/png",
            "sizes": "192x192"
        },
        {
            "src": "logo512.png",
            "type": "image/png",
            "sizes": "512x512"
        }
    ],
    "start_url": ".",
    "display": "standalone",
    "theme_color": "#000000",
    "background_color": "#ffffff"
```

**Review by riginoommen — APPROVED** [review]

**Review by rejahrehim — APPROVED** [review]
